server/lib/handlebars.js

```diff
@@ -3,7 +3,7 @@ import moment from 'moment-timezone';
 import { resizeImage, capitalize, formatCurrencyObject, pluralize } from './utils';
 
 // from https://stackoverflow.com/questions/8853396/logical-operator-in-a-handlebars-js-if-conditional
-handlebars.registerHelper('ifCond', (v1, operator, v2, options) => {
+handlebars.registerHelper('ifCond', function(v1, operator, v2, options) {
```

Comment:
Add to do this because arrow functions have their own context (`this` is reset) thus is was impossible to access template's parameters in a `ifCond`.